js/ui/tree_list/ui.tree_list.data_controller.js

```diff
@@ -184,29 +184,29 @@ exports.DataController = dataControllerModule.controllers.data.inherit((function
         },
 
         /**
-         * @name dxTreeListMethods_loadChildren
-         * @publicName loadChildren()
+         * @name dxTreeListMethods_loadDescendants
+         * @publicName loadDescendants()
          * @return Promise
          */
         /**
-         * @name dxTreeListMethods_loadChildren
-         * @publicName loadChildren(keys)
+         * @name dxTreeListMethods_loadDescendants
+         * @publicName loadDescendants(keys)
          * @param1 keys:array
          * @return Promise
          */
         /**
-         * @name dxTreeListMethods_loadChildren
-         * @publicName loadChildren(keys, deep)
+         * @name dxTreeListMethods_loadDescendants
+         * @publicName loadDescendants(keys, childrenOnly)
          * @param1 keys:array
-         * @param2 deep:boolean
+         * @param2 childrenOnly:boolean
          * @return Promise
          */
-        loadChildren: function(keys, deep) {
+        loadDescendants: function(keys, childrenOnly) {
             if(!this._dataSource) {
                 return;
             }
 
-            return this._dataSource.loadChildren(keys, deep);
+            return this._dataSource.loadDescendants(keys, !childrenOnly);
```

Comment:
Why not using the same parameters logic? If TreeList `loadDescendants` method has the flag that indicates to load only child nodes or load all descendants, it's really strange that dataSource's `loadDescendants` method invert second option logic. It will complicate understanding of code in future.